package.json

```diff
@@ -24,9 +24,9 @@
   },
   "dependencies": {
     "pg": "^6.1.3",
-    "sequelize": "^3.30.2"
+    "sequelize": "3.30.2 - 4.2"
```

Comment:
(I realize you didn't include them both here initially so if you want to discuss this later that's coo with me)